scripts/image

```diff
@@ -273,7 +273,7 @@ if [ "$1" = "release" -o "$1" = "mkimage" -o "$1" = "amlpkg" -o "$1" = "noobs" ]
   )
 
   # Create image files if requested
-  if [[ ( "$1" = "amlpkg" || "$1" = "noobs" || "$1" = "mkimage" ) && -n "$BOOTLOADER" ]]; then
+  if [[ ( "$1" = "noobs" || "$1" = "mkimage" ) && -n "$BOOTLOADER" ]]; then
     # INSTALL_SRC_DIR can be board specific
     if [ -n "$DEVICE" -a -d "$PROJECT_DIR/$PROJECT/devices/$DEVICE/install" ]; then
       INSTALL_SRC_DIR="$PROJECT_DIR/$PROJECT/devices/$DEVICE/install"
```

Comment:
Looks like you can also remove this block that sets `INSTALL_SRC_DIR` (and the passing of it to `scripts/mkimage`), from what I can tell it is only used by amlpkg.

Reply:
Thank you, `amlpkg` drop commit updated.